tuf_conformance/test_basic.py

```diff
@@ -173,22 +178,25 @@ def test_basic_metadata_hash_support(
 
     # Construct repository with hashes in timestamp/snapshot
     repo.compute_metafile_hashes_length = True
-    repo.update_snapshot()  # v2
+    repo.publish([Targets.type])  # v2
+    repo.publish([Snapshot.type, Timestamp.type])
```

Comment:
We should combine these two `repo.publish` calls.

Reply:
https://github.com/theupdateframework/tuf-conformance/pull/188/commits/33284a464e0ae32c93f93da0e8b9c484d552faf7